modules/rosa-sts-aws-requirements-access-req.adoc

```diff
@@ -7,5 +7,5 @@
 
 * Red Hat must have AWS console access to the customer-provided AWS account. Red Hat protects and manages this access.
 * You must not use the AWS account to elevate your permissions within the {product-title} cluster.
-* Actions available in the `rosa` CLI utility or {cluster-manager-url} console must not be directly performed in your AWS account.
+* Actions available in the {product-title} (ROSA) CLI (`rosa`) utility or {cluster-manager-url} console must not be directly performed in your AWS account.
```

Comment:
Line above should have "{product-title} (ROSA)" so that this line can just use "ROSA CLI (`rosa`)

Reply:
To maintain consistency, maybe, "the ROSA CLI " only?